ddl/adminpausetest/ddl_data_generation.go

```diff
@@ -0,0 +1,156 @@
+// Copyright 2022 PingCAP, Inc.
```

Comment:
Can we move this file to other package and public the api？I think it will be very useful for testing

Reply:
That would be fine.